lib/_stream_duplex.js

```diff
@@ -74,7 +74,7 @@ Object.defineProperty(Duplex.prototype, 'writableHighWaterMark', {
   // userland will fail
   enumerable: false,
   get() {
-    return this._writableState.highWaterMark;
+    return this._writableState && this._writableState.highWaterMark;
```

Comment:
I think this should return the `-1` or the default instead of `false`.

Reply:
Thanks, I've changed to the defalut value `16 * 1024`, PTAL.

https://github.com/nodejs/node/blob/09cdc3782488d2e6db09baf75b0dfa1f68e5dc4f/lib/internal/streams/state.js#L22-L23